c7n/resources/kms.py

```diff
@@ -207,3 +218,69 @@ def get_matching_aliases(self, resources, event=None):
                 if self.match(r.get('KeyAlias')):
                     matched.append(r)
         return matched
+
+
+@Key.action_registry.register('remove-statements')
+@KeyAlias.action_registry.register('remove-statements')
+class RemovePolicyStatement(RemovePolicyBase):
+    """Action to remove policy statements from KMS
+
+    :example:
+
+        .. code-block: yaml
+
+           policies:
+              - name: kms-key-cross-account
+                resource: kms-key
+                filters:
+                  - type: cross-account
+                actions:
+                  - type: remove-statements
+                    statement_ids: matched
+    """
+
+    permissions = ('kms:GetKeyPolicy', 'kms:PutKeyPolicy')
+
+    def process(self, resources):
+        results = []
+        client = local_session(self.manager.session_factory).client('kms')
+        for r in resources:
+            key_id = r.get('TargetKeyId', r.get('KeyId'))
+            assert key_id, "Invalid key resources %s" % r
+            try:
+                if self.process_resource(client, r, key_id):
+                    results.append(r)
+            except:
+                self.log.exception(
+                    "Error processing sns:%s", key_id)
+        return results
+
+    def process_resource(self, client, resource, key_id):
+        if 'Policy' not in resource:
+            try:
+                resource['Policy'] = client.get_key_policy(
+                    KeyId=key_id, PolicyName='default')['Policy']
+            except ClientError as e:
+                if e.response['Error']['Code'] != "NotFoundException":
+                    raise
+                resource['Policy'] = None
+
+        if not resource['Policy']:
+            return
+
+        p = json.loads(resource['Policy'])
+        statements, found = self.process_policy(
+            p, resource, CrossAccountAccessFilter.annotation_key)
+
+        if not found:
+            return
+
+        client.put_key_policy(
+            KeyId=key_id,
+            PolicyName='default',
```

Comment:
could you add a link to the key policies or docs or note as comment or doc string that there is only one key policy supported by kms namely the 'default' one. if that changes this will need to be revisited.

http://docs.aws.amazon.com/kms/latest/developerguide/programming-key-policies.html#list-policies